README.md

```diff
@@ -129,6 +129,20 @@ null_ls.setup({
 
 ### In a project
 
+#### `nixfmt-tree`
+
+[`nixfmt-tree`](<<<) provides a zero-setup starter using `treefmt`.
+
+Simply add it to your shell:
+
+```nix
+mkShell {
+  packages = [ pkgs.nixfmt-tree ];
+}
+```
+
+And run `treefmt` to format your entire project.
```

Comment:
Ideally we'd hint that the shell needs to be entered and that this pre-configured treefmt will only touch nix files:

```suggestion
Then run `treefmt` from within your shell to format all nix files in your project.
```

My wording feels slightly awkward, but maybe it inspires something better?